src/d3a/constants.py

```diff
@@ -45,3 +45,6 @@
 CN_PROFILE_EXPANSION_DAYS = 7
 
 RUN_IN_REALTIME = False
+
+
+FEED_IN_TARIFF = 20
```

Comment:
Nit: why two blank lines?
```suggestion

FEED_IN_TARIFF = 20
```